carts/views.py

```diff
@@ -25,4 +25,21 @@ def get(self, request):
                     }for cart_option in cart.cartoption_set.all()],
                 } for cart in carts]
 
-        return JsonResponse({'result' : result}, status=200)
\ No newline at end of file
+        return JsonResponse({'result' : result}, status=200)
+    
+    @login_required
+    def patch(self, request):
+        try:
+            data     = json.loads(request.body)
+            user     = request.user.id
+            cart_id  = data['cart_id']
+            quantity = data["quantity"]
+            cart     = Cart.objects.get(id=cart_id, user_id=user)
+
+            cart.quantity = quantity
+            cart.save()
+
+            return JsonResponse({"message":"SUCCESS"}, status=201)
```

Comment:
```suggestion
            return JsonResponse({"message":"SUCCESS"}, status=200)
```
201은 새로운 resource가 생성 될 때 사용하므로 200이 더 적절합니다.